includes/tracking/class-pixel.php

```diff
@@ -207,5 +222,125 @@ public static function render() {
 		self::track_seen( $newsletter_id, $tracking_id, $email_address );
 		exit;
 	}
+
+	/**
+	 * Schedule log processing.
+	 */
+	public static function schedule_log_processing() {
+		if ( ! \wp_next_scheduled( 'newspack_newsletters_tracking_pixel_process_log' ) ) {
+			\wp_schedule_single_event( time() + 60, 'newspack_newsletters_tracking_pixel_process_log' );
+		}
+	}
+
+	/**
+	 * Log pixel URL.
+	 *
+	 * @param string $url         Pixel URL.
+	 * @param int    $post_id     ID of the newsletter.
+	 * @param string $tracking_id Tracking ID.
+	 * @param string $email_tag   Email address of the recipient.
+	 *
+	 * @return string
+	 */
+	public static function log_pixel_url( $url, $post_id, $tracking_id, $email_tag ) {
+		return \add_query_arg(
+			[
+				'data' => \base64_encode(
```

Comment:
Why is this content base64-encoded?

Reply:
That's actually not required at all. Updated in 2efd68c70f7218c2ccd77d6edbc5d61c12ceb082 and 669a35886aa61c04f4f35a6c287b29f7137b5d2b for simplicity.